src/sass/messaging/partials/_message-compose.scss

```diff
@@ -472,37 +406,77 @@ button.msg-btn-save {
   }
 }
 
-#msg-reply {
-  // Overrides USWDS positioning
-  .usa-input-error {
-    position: static;
-    right: auto;
+// -------- Reply to a message
+.msg-reply-detail {
+  border-bottom: 1px solid $color-gray;
+  padding: 1.5rem;
+
+  label {
+    display: inline-block;
   }
 
-  .msg-compose-error {
-    .msg-write {
-      padding: 1.5rem;
-      margin-left: 0;
+  @media (min-width: $medium-screen) {
+    border: none;
+    padding: 0;
+
+    &:nth-child(1n+2) {
+      display: none;
     }
+  }
+}
+
+@media (min-width: $medium-screen) {
+  .msg-reply-details {
+    border: 1px solid $color-gray;
+    border-bottom: none;
+    cursor: pointer;
+    padding: 1rem;
 
-    .msg-send-group {
-      margin-left: 0;
+    &.opened {
+      .msg-reply-detail {
+        display: block;
+      }
     }
   }
+}
 
-  .msg-field {
-    border-top: none;
+.msg-reply-message {
+  padding-bottom: 0;
+  padding-left: 0;
+  
+  .msg-write label {
+    left: -9999rem;
+    position: absolute;
   }
 
-  .msg-reply-detail {
-    border-bottom: 1px solid $color-gray;
-    padding: 1.5rem;
+  &.msg-compose-error {
+    padding-top: 0;
+  }
+}
 
-    label {
-      display: inline-block;
+.msg-reply-message.msg-compose-error {
+  .msg-write {
+    padding-left: 1rem;
+
+    @media (min-width: $medium-screen) {
+      border-color: $color-gray;
+      border-style: solid;
+      border-width: 1px 1px 0;
     }
   }
+}
 
+.msg-compose-error .usa-input-error {
+  padding-bottom: 1rem;
+  padding-right: 1rem;
```

Comment:
Could be wrong, but I _think_ you intended for this to give the error area some padding in the reply form; it looks like you can accomplish this and resolve the unnecessary padding in the new message form by (1) removing this block and (2) giving 
`.msg-reply-message.msg-compose-error .msg-write` a padding of 1rem instead of just `padding-left: 1rem`.

As a side note, as far as I can tell, this selector only affects the textarea-related components, or more specifically, just the `.msg-write` class. There's this confusing hierarchy of `.msg-write-group.usa-input-error` > `.msg-write` > `.usa-input-error` when that section has an error, so in turn, it can be confusing to tell what the selector is affecting.